lib/pdk/util/template_uri.rb

```diff
@@ -207,11 +207,14 @@ def self.templates(opts)
       end
 
       # @returns String
-      def self.default_template_ref
+      def self.default_template_ref(uri = nil)
```

Comment:
This seems a little complex. Maybe this is better (not sure)?

``` ruby
def self.default_template_ref(uri = nil)
  return 'master' if PDK::Util.development_mode?
  return PDK::TEMPLATE_REF if uri.nil?  # The default url always uses PDK::TEMPLATE_REF as the ref
  uri = new(uri) unless uri.is_a?(self)
  uri.default? ? PDK::TEMPLATE_REF : 'master'
end

Reply:
Yeah I think @glennsarti 's guard clause version is clearer.